src/sentry/snuba/metrics_layer/query.py

```diff
@@ -532,14 +532,24 @@ def convert_snuba_result(
     return snuba_result
 
 
+def fetch_metric_mris_for_use_cases(
```

Comment:
I think you should make this `_fetch_metric_mris_for_use_cases`, to drive home the point that this is not the accepted way to query this data.

Reply:
This is a good point.